src/Core/Resolvers/DWSqlQueryBuilder.cs

```diff
@@ -109,12 +109,12 @@ private static string GenerateColumnsAsJson(SqlQueryStructure structure, bool su
                 {
                     col_value = $"CAST([{col_value}] AS NVARCHAR(MAX))";
                     // Create json. Example: "book.id": 1 would be a sample output.
-                    stringAgg.Append($"\"{escapedLabel}\":\' + ISNULL(STRING_ESCAPE({col_value},'json'),'null') + \'");
+                    stringAgg.Append($"N\'\"{escapedLabel}\":\' + ISNULL(STRING_ESCAPE({col_value},'json'),'null')");
```

Comment:
Is this perf issue applicable to this modification? https://github.com/sequelize/sequelize/issues/3752#issuecomment-334866215

Can you add more details to how you identified `N\'` as a solution?

Reply:
Im not sure if the N' will cause a whole table scan because its just on the key and not the value right? From my understanding the concern is when you would do it on the result returned by the sql query.

Reply:
I confirmed with UC to ensure this would not lead to any perf impacts